policy-service/src/component-tests/java/uk/gov/gchq/palisade/component/policy/model/AuditErrorMessageTest.java

```diff
@@ -23,74 +23,50 @@
 import org.springframework.boot.test.json.ObjectContent;
 import org.springframework.test.context.ContextConfiguration;
 
-import uk.gov.gchq.palisade.Context;
-import uk.gov.gchq.palisade.service.policy.exception.NoSuchPolicyException;
+import uk.gov.gchq.palisade.component.policy.CommonTestData;
 import uk.gov.gchq.palisade.service.policy.model.AuditErrorMessage;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
 
-import static java.util.Map.entry;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
 
 @JsonTest
 @ContextConfiguration(classes = AuditErrorMessageTest.class)
-class AuditErrorMessageTest {
+class AuditErrorMessageTest extends CommonTestData {
 
     @Autowired
     private JacksonTester<AuditErrorMessage> jsonTester;
 
     /**
      * Grouped assertion test
      * Create the object with the builder and then convert to the Json equivalent.
-     * Takes the JSON Object, deserialises and tests against the original Object
+     * Takes the JSON Object, deserializes and tests against the original Object
      *
      * @throws IOException throws if the {@link AuditErrorMessage} object cannot be converted to a JsonContent.
-     *                     This equates to a failure to serialise or deserialise the string.
+     *                     This equates to a failure to serialise or deserialize the string.
      */
     @Test
-    void testGroupedDependantErrorMessageSerialisingAndDeserialising() throws IOException {
-        Context context = new Context().purpose("testContext");
-        Map<String, Object> attributes = new HashMap<>();
-        attributes.put("messagesSent", "23");
+    void testGroupedDependantErrorMessageSerializingAndDeserializing() throws IOException {
 
-        AuditErrorMessage auditErrorMessage = AuditErrorMessage.Builder.create()
```

Comment:
Only a suggestion, could you put the creation of this instance in your CommonTestData

Reply:
This has already been added to the CommonTestData